data/human/human missions.txt

```diff
@@ -7497,3 +7497,211 @@ mission "Adelita 5"
+
+
+
+mission "Syndicate Business 2"
+	name "Purchase bond on <planet>"
+	description "Bring Howard to <destination> in order to finish the paperwork you'll need to finalize your bond agreement with Megaparsec Incorporated."
+	passengers 1
+	source "Sunracer"
+	destination "Hephaestus"
+	to offer
+		has "Syndicate Business 1: done"
+		credits > 1000000
+	on offer
+		conversation
+			`You visit the kiosk location Howard gave you at the spaceport, where he immediately spots and heads toward you. Grateful for saving his life, he thanks you profusely and excitedly tells you what an exclusive offer you've received to be able to purchase a bond of Megaparsec Incorporated.`
+			`	"To reiterate, in order to purchase a perpetual Megaparsec Incorporated bond you will need to provide us with an up-front payment of one million credits, and sign some legal documents. Our promised returns are 20% per annum, which we expect to continue into perpetuity, paid out to you as a daily dividend of 500 credits. Do you accept these terms?"`
+			choice
+				`	"Sure, here's the one million credits."`
+					goto doit
+				`	"No, I won't become an investor."`
+					decline
+			label doit
+			`	After you've signed numerous dotted lines, Howard says, "Alright, we have your payment and preliminary signatures. This particular investment is in partnership with Syndicated Systems, so we'll need to submit this first set of paperwork to their office and then go to <destination> to sign the final documents and conduct a biometric scan at our secure facility. I'll go with you to help finalize the paperwork. Once that is complete, you should immediately start receiving dividend payments."`
+				accept
+	on accept
+		payment -1000000
+	on complete
+		conversation
+			`You land on <destination> and follow Howard's instructions to the biometric scan center. You hand in the first set of paperwork you filled out, and after scanning your fingerprint, retinas, and getting a blood sample, Howard comes over and says, "Excellent, only one last set of paperwork left to sign." He places one more set of documents on the table and taps the stack of papers.`
+			`	"New regulations dictate that we inform you that the proceeds of this bond may be used in high risk activities such as strip mining, clearing endangered habitats, or doing business on worlds with alleged human rights violations. All boilerplate language, of course, but you have to disclose every possibility these days." He shrugs.`
+			`	He then taps the documents the documents again and says, "This is your final decision and set of paperwork. I would also like to disclose that your initial deposit covering the full one million credits is non-refundable. So, as a last formality, would you like to invest in the full investment suite?"`
```

Comment:
```suggestion
			`	He then taps the documents again and says, "This is your final decision and set of paperwork. I would also like to disclose that your initial deposit covering the full one million credits is non-refundable. So, as a last formality, would you like to invest in the full investment suite?"`
```
This is the final decision, but they keep the money regardless of what I choose? Is this satire?

Reply:
Going more for a narrative device than satire, reflecting some of the unfairness of the Syndicate. The feel I'm going for is the discomfort of a sleazy salesman who strings the player along with his charm, paperwork, and fine print in the context of the Syndicate. When investing with the Syndicate, there's minimal consumer rights -- so there's no recourse for refunds or moral qualms after you've signed that first contract. 

By the time you're signing the last paperwork on Hephaestus you've already committed to the investment. Howard's been slickly stringing you through all the documents, and gives you a last minute legal disclaimer that forces you to acknowledge the ethical problems of your investment -- after you've already paid. You _technically_ have the option to back out, but like Howard says, it's really just a formality, and shows how little real choice most people have when dealing with the Syndicate (and in the next missions that theme evolves: as you get richer you get more choices, but they're still dwarfed by the Syndicate's overwhelming size and power).

So the ending bit is a narrative device showing how in the Syndicate, the decisions you make can be trapped in the fine print to pull you towards ethically bad actions, and how you don't have a good way to get out of them. It's (hopefully) an uncomfortable reminder of how little recourse First Last has once Howard has their money, and the 'final decision' is really just a moral dilemma after the fact.

Reply:
I'm not sure not telling me something until I've already committed is "fine print". That's just not telling me.
It's not "last minute". As you've said, I've already committed, I've already paid. Last minute would be if he tole me as I was writing the cheque (metaphorically). The thing is already done.
And, given that the arrangement here is that I am buying a bond, presenting this as a "final decision" when I've already fulfilled my part of the arrangement (give them money) and giving me the option to "back out", which is basically just donating whatever income I would have received back to the Syndicate, doesn't feel "sleezy" or "unfair" as much as it does actual theft.

I get that the Syndicate is a capitalist organisation who doesn't value consumer rights, but I think there is a difference between that and what is presented here. Not telling me something that he is apparently required to mention until after I've done everything he needs of me is just straight up lying.

This kind of just feels like "Syndicate do some bad things, so they'd probably do this bad thing."

Frankly, the manner in which this is presented feels manufactured. If this is something he is required to disclose, it should be before I commit. If he is not required to disclose it, then why is he doing so at all? I have already committed, the second contract just seems like an excuse for him to tell me this additional information for the purpose of creating a moral dillema, even though the contract doesn't really do anything (since I already paid). Did the original documents just not make any reference to what I would receive in exchange for my money? Did I just give them money with no contract on what I would get in exchange?